src/IoHandlers.h

```diff
@@ -29,6 +29,8 @@
 #ifndef IOHANDLERS_H_
 #define IOHANDLERS_H_
 #include "common.h"
+#include "Message.h"
+#include "Defs.h"
```

Comment:
no need. I used them while testing a specific scenario and forget to remove them.